internal/tiltfile/k8s.go

```diff
@@ -219,9 +274,9 @@ func (s *tiltfileState) k8sResource(thread *starlark.Thread, fn *starlark.Builti
 		return nil, fmt.Errorf("image arg must be a string or fast_build; got %T", imageVal)
 	}
 
-	portForwards, err := s.convertPortForwards(name, portForwardsVal)
+	portForwards, err := convertPortForwards(portForwardsVal)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "%s %q", fn.Name(), name)
```

Comment:
i thought starklark printed the function name by default? not sure why this one is a special case

Reply:
It does in the case of failed `UnpackArgs`, but not in other cases!

e.g.,
```
k8s_resource('hello-world', image='.sadf')
```

```
Traceback (most recent call last):
  /Users/matt/go/src/github.com/windmilleng/servantes/Tiltfile:114: 
<toplevel>
Error: invalid reference format
```